.github/workflows/windows.yml

```
@@ -61,6 +61,14 @@ jobs:
       with:
         languages: 'cpp'
 
+    - name: Cache the build folder
+      uses: actions/cache@3624ceb22c1c5a301c8db4169662070a689d9ea8
+      env:
+        cache-name: cache-nuget-modules
+      with:
+        path: build
+        key: ${{ matrix.platform }}-${{ matrix.arch }}-${{ hashFiles('**/CMakeLists.txt') }}-${{ hashFiles('.github/workflows/windows.yml') }}
```

actionlint

property "platform" is not defined in object type {}

(expression)